droidsx/droid-bodies
# Enable graphite support in WSL

### Summary

Added Homebrew and Graphite CLI installation to the development container.

### What changed?

The Dockerfile for the development container now includes steps to install Homebrew and the Graphite CLI. This is done using the official Homebrew installation script, followed by adding Homebrew to the PATH and installing the Graphite CLI using Homebrew.

### What to test?

1. Build the development container using the updated Dockerfile.
2. Ensure that your current Graphite workflow remains unaffected. This change is intended only for WSL users.

### Why this change is required?

This change enables Windows developers to use the Graphite CLI within the development container.
